third_party/move/move-model/src/model.rs

```diff
@@ -637,11 +644,21 @@ impl GlobalEnv {
         self.generated_by_v2 = yes
     }
 
-    /// Returns if this is generated by v2.
+    /// Returns true if this is generated by v2.
     pub fn generated_by_v2(&self) -> bool {
         self.generated_by_v2
     }
 
+    /// Sets the language version.
+    pub fn set_language_version(&mut self, version: LanguageVersion) {
+        self.language_version = version
+    }
+
+    /// Returns the language version
```

Comment:
nit: same - no need to comment functions with `self commenting` names

Reply:
dito